sdk/attestation/azure-security-attestation/README.md

```diff
@@ -1,39 +1,362 @@
 # Azure Attestation Package client library for C++
 
-Azure Template Package client library for C++ (`azure-security-attestation`)  matches necessary patterns that the development team has established to create a unified SDK written in the C++ programming language. These libraries follow the Azure SDK Design Guidelines for C++.
+Microsoft Azure Attestation (preview) is a unified solution for remotely verifying the trustworthiness of a platform and integrity 
+of the binaries running inside it. The service supports attestation of the platforms backed by Trusted Platform Modules (TPMs) 
```

Comment:
Editors which don't implement line wrap reasonably (like Visual Studio) turn a single line paragraph into an unreadable mess.

Manually inserting line breaks makes the text readable.

Reply:
That's true, but what line length do we maintain here? It could make edits more difficult, to maintain the line wrapping manually. What about other editors like VS Code? If I add a few words, how do I fixup the manual line breaks? Does it become a chain of edits?